TypeScript problem: Property className does not belong to the Props type definition

I'm using your library in a React-TypeScript project. When adding a `className` attribute to the `NanoClamp` tag, the TypeScript compiler complains about the unknown property `className`.

Code:
```jsx
<NanoClamp text={s.value} lines={2} className="custom-class" />
```

TypeScript error:
```
src/App.tsx:195:61 - error TS2322: Type '{ text: string; lines: number; className: string; }' is not assignable to type 'IntrinsicAttributes & Props'.
  Property 'className' does not exist on type 'IntrinsicAttributes & Props'.
```

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,6 +1,6 @@
 import { createElement, useRef, useEffect, useCallback, useMemo } from 'react'
 
-interface Props {
+interface Props extends React.HTMLAttributes<HTMLElement> {
   accessibility?: boolean
   debounce?: number
   ellipsis?: string
@@ -36,7 +36,7 @@ const NanoClamp = ({
   lines = 3,
   text,
   ...props
-}: Props): JSX.Element | null => {
+}: Props): React.ReactElement | null => {
   const elementRef = useRef<HTMLElement>(null)
   const textRef = useRef<string>(DEFAULT_TEXT)
 

